### PR TITLE
Answer:31

### PR DESCRIPTION
**Pull request by BruceArmstrong007**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

  Converted Angular Project to Standalone (NgModule -> Standalone)

### CODE REVIEW

File: libs/module-to-standalone/admin/feature/src/lib/admin-feature.routes.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Route } from '@angular/router';
+
+export const adminRoutes: Route[] = [
+  {
+    path: '',
+    loadChildren: () =>
+      import('./dashboard/dashboard.routes').then((m) => m.dashboardRoutes),
```
Comment (tomalaforge) [inline libs/module-to-standalone/admin/feature/src/lib/admin-feature.routes.ts:7]: you can use the default keyword to avoid `then`inside your promise

**Comment by tomalaforge** [comment]
Very good 👍 Did you try to use the angular schematic for this ?

File: libs/module-to-standalone/admin/feature/src/lib/admin-feature.routes.ts (hunk 1 above)
Comment (BruceArmstrong007) [reply libs/module-to-standalone/admin/feature/src/lib/admin-feature.routes.ts:7]: ya that will be much cleaner as well, thanks i will implement it from now

**Comment by BruceArmstrong007** [comment]
> Very good 👍 Did you try to use the angular schematic for this ?

@tomalaforge  yes, i do use nx angular schematic for generating components, pipes and directives and stuff

**Pull request closed without merging**